GenderPayGap.WebUI/Classes/IWebTracker.cs

```diff
@@ -1,12 +0,0 @@
-﻿using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
-
-namespace GenderPayGap.Core.Interfaces
-{
-    public interface IWebTracker
-    {
-
-        Task TrackPageViewAsync(Controller controller, string pageTitle = null, string pageUrl = null);
-
-    }

```

Comment:
This has been moved into the `GoogleAnalyticsTracker.cs` file (because it's only used there)